sorc/checkout.sh

```diff
@@ -21,24 +24,24 @@ do
 done
 
 topdir=$(pwd)
-echo $topdir
+logdir="${topdir}/logs"
+mkdir -p ${logdir}
 
-echo fv3gfs checkout ...
-if [[ ! -d fv3gfs.fd ]] ; then
-    rm -f ${topdir}/checkout-fv3gfs.log
-    git clone https://github.com/ufs-community/ufs-weather-model fv3gfs.fd >> ${topdir}/checkout-fv3gfs.log 2>&1
-    cd fv3gfs.fd
-    git checkout 9350745855aebe0790813e0ed2ba5ad680e3f75c
+echo ufs-weather-model checkout ...
+if [[ ! -d ufs_model.fd ]] ; then
+    git clone https://github.com/ufs-community/ufs-weather-model ufs_model.fd >> ${logdir}/checkout-ufs_model.log 2>&1
+    cd ufs_model.fd
+    git checkout ${ufs_model_hash:-22613e8}
     git submodule update --init --recursive
     cd ${topdir}
 else
-    echo 'Skip.  Directory fv3gfs.fd already exists.'
-fi
+    echo 'Skip.  Directory ufs.fd already exists.'
```

Comment:
Should be "ufs_model.fd already exists."

Reply:
Yes, it should. Fixed.